### ISSUE TITLE
Login scopes and responsive issues on mobile.

### ISSUE THREAD

**Issue by luis901101** [medplum/medplum#4043]
Check the following Screenrecord, tested with iPhone 12 mini, iOS 17.3.1:

https://github.com/medplum/medplum/assets/28322469/30cffc18-8af9-48b8-a43b-b2b1933f401d

## What's happening
When login on Medplum App from a mobile:
1. The login form doesn't fit well in the screen, the user has to scroll horizontally to get to the **Next** and **Sign in** buttons.
2. The login shows a last step screen after user password, asking to **Choose scope** and a list of pre-checked checkboxes.

## What I expect to happen
1. The login form should fit the mobile screen so the user doesn't have to scroll horizontally to get to the buttons.
2. Scopes should not be asked to confirm if they are already set on Authentication request, like this request:
```dart
GET https://api.medplum.com/oauth2/authorize?
    response_type=code&
    client_id=XXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXX&
    redirect_uri=https://my.app.com/redirect&
    state=XXXXXXXXXXXX&
    scope=offline_access openid email phone profile
```

**Comment by codyebberson** [comment]
Confirmed, we'll get you a fix soon.

**Comment by luis901101** [comment]
Hi @codyebberson thanks for the feedback.
- Regarding the Scopes UI, understood, I think it would be good having that setting on `ClientApplication` so there is no need to make any changes on auth request.
- Regarding the responsiveness on the login UI it has improved but still has some unexpected behavior. When the user taps on the InputField there is a ZoomIn that causes some parts of the form to be cropped so the **"Next"** button looks cropped and again the user has to scroll horizontally, the same happens in the Password screen if the user wants to tap on the Show/Hide password, it's mandatory to scroll horizontally. Check the following Screenrecord.

https://github.com/medplum/medplum/assets/28322469/59391431-883d-4467-b8c1-5a68266b6bd9

**Comment by luis901101** [comment]
@codyebberson you're right, changing from current `maximum-scale=5` to `maximum-scale=1` does the trick, I just tested it, so with the meta tag you proposed is enough:

```html
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1" />
```

by the way just with this would be enough:
```html
<meta name="viewport" content="user-scalable=no,initial-scale=1,maximum-scale=1">
```

**Comment by luis901101** [comment]
Hi @codyebberson any progress on this? Last time we agreed the responsive behavior only misses that minor change in the `viewport`, but what about the login scopes, is it going to be a `queryParam` or a conf in the `ClientApplication`?

**Comment by luis901101** [comment]
Hi @codyebberson is this going to be fixed any time soon? 

1. The responsive behavior in mobile screens still misses the fix we agree above:
```html
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1" />
```

2. And regarding the login scopes? You mentioned above some option to override the requirement like a query parameter in the url or a configuration setting on the `ClientApplication`, any of those approaches works good for me _(preferably the `ClientApplication` setting to let the authorization request unchanged)_, **another way could be to keep the scopes confirmation screen for the user to grant it as **SMART-on-FHIR** mandates but at least showing each scope with a user friendly text like Google and other IDP does with their users for instance:**

:white_check_mark: Allow to see my profile, like name, birthdate and picture (profile)
:white_check_mark: Allow to see my email (email)
:white_check_mark: Allow to see my phone (phone)
:white_check_mark: Remember me (offline_access)

Instead of what currently shows up to the user which not necessarily understands those terms:
<img width="462" alt="scopes" src="https://github.com/medplum/medplum/assets/28322469/d3ea13f0-1a2f-4aaa-8088-ce61af5ec99c">
